packages/apps/api/src/handlers/user.ts

```diff
@@ -216,3 +216,40 @@ export async function changePassword(sierraClient: SierraClient, auth0Client: Au
 
   response.status(200).json(toUser(sierraUpdate.result, auth0Update.result));
 }
+
+export async function searchUsers(auth0Client: Auth0Client, request: Request, response: Response): Promise<void> {
+
+  const page: number = Number(request.query.page);
+  const pageSize: number = Number(request.query.pageSize);
+  const sort: string = request.query.sort as string;
+  const sortDir: number = Number(request.query.sortDir);
+  const query: string = request.query.query as string;
+  if (isNaN(page) || isNaN(pageSize) || !isNonBlank(query) || !isNonBlank(sort) || isNaN(sortDir)) {
+    response.status(400).json(toMessage("All fields must be provided and non-blank"));
+    return;
+  }
+
+  if (!Auth0SearchSortFields.get(sort)) {
+    response.status(400).json(toMessage("Invalid sort field, one of [" + Array.from(Auth0SearchSortFields.keys()) + "] must be provided"));
+    return;
+  }
+
+  if (sortDir !== 1 && sortDir !== -1) {
+    response.status(400).json(toMessage('\'sortDir\' must be \'1\' or \'-1\' to sort ascending and descending respectively'))
```

Comment:
You switched from double quotes to single quotes here, which is odd since it means you now have to escape the single quotes that are part of the string.

Reply:
I've been trying to consistently to use single quotes in the TypeScript, even if that means escaping embedded quotes in some places. I've gone through and switched everything to single quotes where I overlooked this before.

Reply:
Cool, don't mind the quote style particularly, just noticed that it had switched midway.